sys/include/net/gnrc.h

```diff
@@ -144,9 +144,10 @@
  * }
  * ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  *
- * @note It is necessary to react with a reply message to the other thread for
- *       @ref GNRC_NETAPI_MSG_TYPE_SET and @ref GNRC_NETAPI_MSG_TYPE_GET
- *       commands
+ * @note When receiving a message of type @ref GNRC_NETAPI_MSG_TYPE_SET or
+ *       @ref GNRC_NETAPI_MSG_TYPE_GET, it is necessary to acknowledge it by
+ *       calling @ref msg_reply() with a message of type
+ *       @ref GNRC_NETAPI_MSG_TYPE_ACK.
```

Comment:
The value that is at `gnrc_netapi_opt_t::data` (which in turn is pointed to by `msg.content.ptr` of the GNRC_NETAPI_MSG_TYPE_GET msg) on reply. Since that is a pointer this information needs to be provided to the sender somehow (gnrc_netapi_opt_t::data_len only provides the replier with the *maximum space* available at `gnrc_netapi_opt_t::data`).